__tests__/app.test.js

```diff
@@ -189,6 +189,95 @@ describe("/api/articles/:article_id", () => {
             });
         });
     });
+    describe("PATCH", () => {
+        test("Returns 'Status: 204' with empty object if sent empty request body", () => {
+            return request(app).patch('/api/articles/1')
+            .send()
+            .expect(204)
+            .then(({ body }) => {
+                expect(body).toEqual({})
+            });
+        });
+        test("Returns 'Status: 200' with an article object", () => {
+            return request(app).patch('/api/articles/1')
+            .send({ inc_votes: 1 })
+            .expect(200)
+            .then(({ body }) => {
+                expect(typeof body.updatedArticle).toBe("object");
+                expect(Array.isArray(body.updatedArticle)).toBe(false)
+            });
+        });
+        test("Returns correct article object for ID with expected keys", () => {
+            return request(app).patch('/api/articles/1')
+            .send({ inc_votes: 1 })
+            .expect(200)
+            .then(({ body }) => {
+                const updatedArticle = body.updatedArticle;
+                expect(updatedArticle).toHaveProperty("article_id", 1)
+                expect(updatedArticle).toHaveProperty("title")
+                expect(updatedArticle).toHaveProperty("topic")
+                expect(updatedArticle).toHaveProperty("author")
+                expect(updatedArticle).toHaveProperty("body")
+                expect(updatedArticle).toHaveProperty("created_at")
+                expect(updatedArticle).toHaveProperty("votes")
+                expect(updatedArticle).toHaveProperty("article_img_url")
+            });
+        });
+        test("Returns article object with 'votes' key updated by correct amount, including by negative numbers", () => {
+            return request(app).patch('/api/articles/1')
+            .send({ inc_votes: -20 })
+            .expect(200)
+            .then(({ body }) => {
+                const updatedArticle = body.updatedArticle;
+                expect(updatedArticle).toHaveProperty("article_id", 1)
+                expect(updatedArticle).toHaveProperty("votes", 80)
+            });
+        });
+        test("Updates to 'votes' column of the article are reflected in the articles database", () => {
```

Comment:
This test, is not really required as you are testing the returned value from a successful patch in the above test. Your SQL query returns the updated article as it looks after the update the same as this query in the test does.